source/includes/_configuration-and-environments.md

```diff
@@ -0,0 +1,81 @@
+# Configuration and environments
+One of the goals of booster is to avoid configuration parameters as much as
+possible, inferring everything automatically from your code. However, there
+are some aspects that can't be inferred (like the application name), or it
+is useful to change them under certain circumstances, especially when using
+different [environments](#environments)
+
+## Booster configuration
```

Comment:
I'd add some clarification before throwing the code example:

```suggestion
## Booster configuration

Booster supports multiple configuration environments (i.e: You can have a file named `src/config/production.ts` file with the configuration for your production environment and a file named `src/config/pre-production.ts` with a shared stage for QA). Let's see an example:
```

Reply:
Ok, but I would change this a bit to leave clear that the name of the file is not necessarily the name of the environment, but the name you pass to the first parameter. I will improve the doc to leave this distinction clear